app/uk/gov/hmrc/economiccrimelevyaccount/models/DocumentDetails.scala

```diff
@@ -105,7 +105,6 @@ case class DocumentDetails(
           case NewCharge | AmendedCharge => StandardPayment
           case InterestCharge            => Interest
           case Payment                   => Overpayment
-          case _                         => Unknown
```

Comment:
Will this crash if this catch all isn't triggered on an unknown type?

Reply:
In the `FinancialDataDocumentType` `read` method we are never deserialising to any Unknown type, only into these 4 types (ref. FinancialDataDocumentType lines 32-43). `Case _` will never trigger